astropy/io/misc/asdf/__init__.py

```diff
@@ -0,0 +1,10 @@
+import warnings
+
+from astropy.utils.exceptions import AstropyDeprecationWarning
+
+warnings.warn(
+    "ASDF functionality for astropy is being moved out of the astropy"
+    " package to the new asdf-astropy package. Please use this package"
+    " instead.",
```

Comment:
```suggestion
    " instead. ``astropy.io.misc.asdf`` is deprecated since astropy 5.1 and will be removed in a future release.",
```

Reply:
Do the double backticks render in a warning message?  I thought that was just an RST thing.

Reply:
Oh, ops! I wasn't thinking.